### DISCUSSION TITLE
Objects won't appear on site

### DISCUSSION THREAD

**Discussion by Aeppers1** [CollectionBuilder/collectionbuilder.github.io#65] [Q&A]
I have gone through the tutorials again and again, and I've only been able to get the changes I made to the URL, base URL, title, tagline, and description variables in the config.yml to  show on my site. I have 22 jpgs uploaded to the objects folder (after changing to jpgs from pdgs), I have uploaded a csv to the data folder with the required elements filled out. I re-uploaded my objects after adjusting size, I cannot get anything to actually show up on my site: shows that I have 0 of 0 objects and there is no data connected.
I would appreciate any insights!

**Comment by evanwill** [comment]
@Aeppers1 the issue is with the column names of your metadata CSV--the required fields have to match the template *exactly* (with no extra spaces or capitalization)!
So in your file ["_data/puzzles.csv"](https://github.com/Aeppers1/puzzleswap/blob/main/_data/puzzles.csv) the first row is currently:

`Object ID,File Name,Title,Format,Brand/Maker,Pieces,Dimensions,Diagram Included,Materials,Age,Created,Notes`

For example, notice "Object ID" has capitalization and a space--for the template to show any objects this column must be named `objectid`. So to fix your issues with objects you will need to replace that first row with this instead:

`objectid,filename,title,format,Brand/Maker,Pieces,Dimensions,Diagram Included,Materials,Age,Created,Notes`

Making that update will get the items correctly displaying through out the site. The next step would be to visit the various [config- CSVs to configure the pages to display your custom fields](https://collectionbuilder.github.io/cb-docs/docs/customization/). 

One other issue you have right now is in ["_data/config-nav.csv"](https://github.com/Aeppers1/puzzleswap/blob/main/_data/config-nav.csv) there is three empty lines in the middle of the file--delete those blank lines to avoid weird empty links in your navbar!

Good luck!

**Reply by Aeppers1** [reply]
Okay, does that mean that the titles of each individual item should also be listed as lowercase?

Also, thank you! So, so much. I have created and deleted about 5 different entire repositories trying to get things to work.

**Reply by dcnb** [reply]
Titles can be any case you'd like!

It looks like you're still having some issues with the fields being upper and lowercase. For instance, your config-metadata.csv file lists the fields you'd like to display in lowercase. That's good, but it has to match one-to-one with your _data/puzzles.csv

Try replacing the contents of that file so that it looks like the below: 

field,display_name,browse_link,external_link
title,Title,,
format,Format,,
Brand/Maker,Brand/Maker,,
Pieces,Pieces,,
Dimensions,Dimensions,,
Diagram,Diagram Included,, 
Materials,Materials,, 
Age,Age,,
Created,Created,,
Notes,Notes,,

Best practice for this type of work is to make all of your fields lowercase and list them throughout the site as such, but if the field names are capitalized in the main metadata csv -- for you, that's puzzles.csv -- then you just need to make sure they match in the various config-.csv files. You match the fields correctly in the config-browse.csv file for instance. 

It's just about making everything match exactly. The code can't read any context; it's just looking for matches. 

Good luck!

**Reply by Aeppers1** [reply]
Thank you!
